README.md

```diff
@@ -23,8 +23,30 @@ Make sure you have installed all of the following prerequisites on your machine:
 
 
 ## Quick Start
+Clone repo: 
 
 ```
+$ git clone https://github.com/nspcc-dev/neofs-dev-env.git
+```
+
+Run next commands from project's root:
+```
+$ make get
```

Comment:
`get` will be called on `up`. There is not much sense in doing it beforehand.

Reply:
`make hosts` will not be executed without `make get`